scripts/build/redhat/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "etcher-electron",
+  "productName": "Etcher",
+  "genericName": "Etcher",
+  "homepage": "https://etcher.io/",
+  "icon": "assets/icon.png",
```

Comment:
Nope adding lower resolution icons doesnt help either. must be related to repository only download...